tests/test_image_transformer.py

```diff
@@ -1,93 +1,47 @@
-import itertools
 import datetime
+import itertools
 import json
 import logging
-import os.path
 import time
-import tempfile
 import unittest
 from structlog import wrap_logger
-
-from reportlab.lib.pagesizes import A4
-from reportlab.platypus import SimpleDocTemplate
-
 from transform.transformers.ImageTransformer import ImageTransformer
-from transform.transformers.PDFTransformer import PDFTransformer
-
-log = wrap_logger(logging.getLogger(__name__))
 
 
 class ImageTransformTests(unittest.TestCase):
 
     def setUp(self):
-        self.max_diff, self.maxDiff = self.maxDiff, None
-
-    def tearDown(self):
-        self.maxDiff = self.max_diff
-
-    def test_image_index_date(self):
-
-        with open("./tests/csv/valid.023.0205.json") as fb:
+        with open("./tests/data/eq-mwss.json") as fb:
             data = fb.read()
-        reply = json.loads(data)
+        self.reply = json.loads(data)
 
-        with open("./transform/surveys/023.0205.json") as fb:
```

Comment:
Do we want to have a test with different data also, 023.0205 & 134.0005 ?

Reply:
These are the tests that were in place for the existing image transformer . I think if we have missing surveys we should definitely add them . I am not sure if it should be part of this ticket though ?